.changeset/khaki-experts-pretend.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
🔧 This should be a `minor` version bump. @nikosdouvlis Should we mention that we dropped the experimental prop?

Reply:
Agree with @dimkl here, we might need to mention this one